source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -53,8 +53,15 @@ This video is a nice introduction into the world of Ember Octane and Glimmer com
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Empress: the Ember Press Episode 129](https://emberweekend.com/episodes/empress-the-ember-press)
+
+Episode 129 of the Ember press featured [@mansona](https://github.com/mansona) the creator of Authmaker. This episode included a range of topics including the migration of moving the Ember guides from Middle man to be powered by Ember. Other areas include Middleman shortcomings and the rework of the guides.
```

Comment:
We use a language linting tool on the blog called [alex]() which helps us to catch insensitive language during our CI run with Travis. Currently this PR fails the language check with the following error message:

```
source/2019-03-30-the-ember-times-issue-91.md
  58:257-58:266  warning  `Middleman` may be insensitive, use `Intermediary`, `Go-between` instead  middleman-middlewoman  retext-equality
```
You can also find the [full log over here](https://travis-ci.org/ember-learn/ember-blog/builds/512893636?utm_source=github_status&utm_medium=notification).

You should be able to allow the usage of the term `Middleman` by applying an exception rule. You can read more about it [in our contribution guides as well](https://github.com/ember-learn/ember-blog/blob/master/source/CONTRIBUTING.md#language-checking-with-alex)

```suggestion
<!--alex ignore middleman-middlewoman-->
Episode 129 of the Ember press featured [@mansona](https://github.com/mansona) the creator of Authmaker. This episode included a range of topics including the migration of moving the Ember guides from Middleman to be powered by Ember. Other areas include Middleman shortcomings and the rework of the guides.
```